server/monitor/src/main/resources/org/apache/accumulo/monitor/resources/external/datatables/css/jquery.dataTables.css

```diff
@@ -0,0 +1,448 @@
+/*
+ * Table styles
```

Comment:
Is this vanilla, or did you have to tweak it? Things in the external folder should correspond to an upstream, so I'm just checking. If we have to tweak it, we can tweak it in our own CSS file or inline.

Reply:
It is vanilla, downloaded as-is from the site.